apps/api/src/app/channeling/entities/channeling_session.entity.ts

```diff
@@ -0,0 +1,10 @@
+import { Schema } from 'mongoose';
+
+export const ChannelingSessionEntity = new Schema({
+  channelingSessionId: String,
+  channelingDoctorId: String,
+  channelingDoctorName: String,
```

Comment:
Doctor name should not be hardcoded into each document. It should be queried using doctorId